metagen_tools/tablegen.m

```diff
@@ -0,0 +1,130 @@
+
+function output = tablegen (basepath2)
```

Comment:
For cohesion with rest of repo, reference to basepath should be 'basepath'. Remove the '2'.